SPECS/trace-cmd/trace-cmd.spec

```
@@ -0,0 +1,108 @@
+# git tag
+#%%global git_commit trace-cmd-v2.6.2
+#%%global git_commit 57371aaa2f469d0ba15fd85276deca7bfdd7ce36
+
+%global srcversion 3.2
+
+Name:          trace-cmd
+Version:       %{srcversion}
```

I recall that the auto-upgarde pipeline used to have issues with a macro in the version.
I hope that is not the case now.
/cc @Nicogbg
```suggestion
Version:       %{srcversion}
```

---

Synced with Nicolas offline, will not use a macro in the version. Updated trace-cmd.spec file